searcher_client/src/lib.rs

```diff
@@ -54,3 +66,62 @@ pub async fn create_grpc_channel(url: &str) -> BlockEngineConnectionResult<Chann
     }
     Ok(endpoint.connect().await?)
 }
+
+pub async fn send_bundle_with_confirmation(
```

Comment:
I like this idea, how about using blocksubscribe and ensure the txs are contiguous to ensure it truly landed as a bundle